assembl/lib/sentry.py

```diff
@@ -0,0 +1,21 @@
+"""Abstract the existence and use of Sentry"""
+from traceback import print_exc
+import sentry_sdk
+
+from assembl.lib.config import get
+
+
+def is_sentry_enabled():
+    return bool(get('sentry_dsn', ''))
+
+
+def capture_message(*args, **kwargs):
+    if is_sentry_enabled():
+        sentry_sdk.capture_message(*args, **kwargs)
+
+
+def capture_exception(*args, **kwargs):
```

Comment:
args and kwargs are not used here (where is the error/exception here?). I think you have to get the POST/GET from the request object (filter emails and passwords)

Reply:
oh yes, I should pass args and kwargs to `capture_exception` like it was before. If exception is not given to `sentry_sdk.capture_exception`, it is taken from `sys.exc_info()`.
About adding the POST/GET from the request, I think we should add those as breadcrumbs but I would like to do it in another PR so we can benefit from the update asap

Reply:
yes :+1:  then, you have to create a ticket :)